common/constants/big.go

```diff
@@ -12,12 +12,25 @@ var (
 	Big0 = big.NewInt(0)
 	// Big1 is 1 represented as type "math/big"
 	Big1 = big.NewInt(1)
+	// Big2Exp232 is 2^232 respresented as type "math/big"
+	Big2Exp232 = util.BigExp(2, 232)
 	// Big2Exp256 is 2^256 respresented as type "math/big"
 	Big2Exp256 = util.BigExp(2, 256)
 )
 
 // Commonly used max values represented as type "math/big"
 var (
+	// MaxUint256 is the maximum uint232 number represented as type "math/big"
```

Comment:
yah so here's the thing.... The tests will only pass if the max target is set to MaxUint256 because of this guy [link](https://github.com/ubclaunchpad/cumulus/pull/112/files#diff-3a5c8ea459d421751fb70343be214598R220)